x/concentrated-liquidity/position_test.go

```diff
@@ -642,7 +660,15 @@ func (s *KeeperTestSuite) TestDeletePosition() {
 				s.Require().Nil(positionIdBytes)
 			}
 
-			err = s.App.ConcentratedLiquidityKeeper.DeletePosition(s.Ctx, test.positionId, s.TestAccs[0], defaultPoolId)
+			if test.creator != nil {
+				defaultCreator = secondaryAccount
+			}
+
+			if test.poolId != 0 && test.poolId != defaultPoolId {
+				defaultPoolId = test.poolId
+			}
```

Comment:
Consider setting `tc.poolId` to the desired value in the test case instead of modifying here